opencart 1.5.6.4/upload/admin/controller/module/smaily_for_opencart.php

```diff
@@ -99,6 +101,30 @@ public function index() {
         $this->data['rss_feed_title'] = $this->language->get('rss_feed_title');
         $this->data['rss_feed_text'] = $this->language->get('rss_feed_text');
         $this->data['rss_feed_url'] = $this->config->get('config_url') . 'index.php?route=smailyforopencart/rss';
+
+        // Layout text and buttons
+        $this->data['table_header_layout_text'] = $this->language->get('table_header_layout_text');
+        $this->data['table_header_position_text'] = $this->language->get('table_header_position_text');
+        $this->data['table_header_status_text'] = $this->language->get('table_header_status_text');
+        $this->data['table_header_sort_order_text'] = $this->language->get('table_header_sort_order_text');
+        $this->data['optin_form_position_top_text'] = $this->language->get('optin_form_position_top_text');
+        $this->data['optin_form_position_bottom_text'] = $this->language->get('optin_form_position_bottom_text');
+        $this->data['optin_form_position_left_text'] = $this->language->get('optin_form_position_left_text');
+        $this->data['optin_form_position_right_text'] = $this->language->get('optin_form_position_right_text');
+        $this->data['button_add_module_text'] = $this->language->get('button_add_module_text');
+        $this->data['button_remove'] = $this->language->get('button_remove');
```

Comment:
You have updated the variable name in translations file `button_remove_module_text`, but have left it unchanged here.